gemfiles/rails6.0.gemfile

```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+
+gemspec path: '../'
+
+gem 'activerecord', '~> 6.0'
+gem 'actionpack', '~> 6.0'
+gem 'mysql2', '>= 0.4.4', '< 0.6.0', platforms: :ruby
```

Comment:
```suggestion
gem 'mysql2', '>= 0.4.4', platforms: :ruby
```

We need the `gem "mysql2"` lines because ActiveRecord doesn’t have an explicit dependency on the gem – only when the DB adapter is required is the mysql2 gem being required. So we usually use the same line as is in the adapter: https://github.com/rails/rails/blob/v6.0.0/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb#L6